### DISCUSSION TITLE
【随波呼吸】帮助了解和练习呼吸

### DISCUSSION THREAD

**Discussion by jason4wang** [creatorsand/.github#91] [Ideas]
**一句话 IDEA：**
帮助了解与练习呼吸，掌握不同呼吸方式的作用，养成良好的呼吸习惯，获得更健康与美好的生活。

**我的角色：**
产品+设计+开发+测试+运营

**你在项目中的具体角色和负责的任务：**
目前依然只是一人项目，所以我将会负责全部任务，包括产品的前期市场调研、需求分析、功能设计、UI设计、功能开发、功能测试及产品上线。

**项目需要的其他角色或技能，发起人以外的团队成员需求：**
如果你刚好对呼吸、正念、冥想等话题感兴趣，又或者你刚好了解过一些我在做的事情，并且有关于产品、设计或营销方面的想法想与我交流，请通过社群与我联系，我相信我们会聊得愉快~

**社群昵称：**
Jason 佳成

**个人简介：**
【产品经理】我的本质工作是产品经理，日常在做的主要是营销方向的小程序产品，有过一些用户增长的产品实践
【有开发经验，但不多】我有自己开发过2个微信小程序，但是基本不涉及到后端技术和网络请求，对于SwiftUI我通过AI+自学，能实现一些不复杂的前端交互
【上海】我居住在上海，如果你想与我聊一聊，欢迎通过社群与我联系

**产品是给谁用的？**
对于已经没有呼吸的人，要说一声抱歉了，可能帮不到你们了。

认真讲！

一个婴儿诞下，他可以不会哭喊，但是他无法不学会吮吸母乳与呼吸。吮吸母乳（或奶瓶）这件事在他2-3岁的时候他可能就不再需要了，但是呼吸将会伴随他直到生命的终了。在成长的过程中，他可能习得的是一些不好的呼吸方式，例如口呼吸、浅而快的呼吸等，这些呼吸方式将会以每天大概2万的次数重复，给他的身体逐年累月地造成负担甚至是伤害。
又或者是一位找寻身心灵慰藉的人，在屡次三番地对冥想或正念跃跃欲试，而后不得其门而入，倍感挫折。
又或者是一位想要稍微了解下呼吸这件事的人，在某个搜索框里输入深呼吸三个字之后得到几十种奇奇怪怪名字的呼吸方式的人，跟着其中一种练习了10分钟，发现似乎没有任何改变、更不知道自己在做什么的人。

我希望能够帮助到那些可能有不良呼吸习惯的人、希望通过呼吸来改善自己的身体或精神状态的人、在冥想或正念中苦苦挣扎而不得其法的人，希望他们能够通过正确呼吸这件并不难的事情，获得更加健康而美好的生活。

**其他产品有什么痛点，用户为什么要选择你的产品？**
首先我想对“其他产品”下一个定义，对于要了解和练习呼吸，我们有太多方式了，例如YouTube、B站、小红书、App、小程序、播客、书籍、线下or线上课程、又或者是一些图文教程，而不仅仅是其他App。

有一个表述是“像呼吸一样简单”，但是如果我们尝试通过其他产品了解呼吸，会发现呼吸这件事原来如此地难。举个例子，主流的观点会说到我们应该使用腹式呼吸，而之所以要这么做的原因是因为腹式呼吸能够帮助我们更加放松身体，但是为什么腹式呼吸能够帮助我们更加放松身体呢？其他产品很少解释。我希望我能够帮助用户用简单的方式从原理上了解呼吸这件事，进而知道每次吸气、呼气或停顿的时候，我们的身体究竟在发生什么。进而能够知道，能够找到的纷杂的呼吸方式ABCDEFG本质上并没有区别，不用纠结自己的练习方式对不对，到底哪个方式好。如果我们不知道答案还要坚持批改试卷，这样的行为看起来就像是掷骰子。

呼吸练习是一件需要至少坚持一段时间才能兑现价值的事情，如果在初期的时候只知道how，不知道why和what，其实是很难坚持下来的。知道how可能能够让人走得更快，但是知道why大概率能让人走得更远。

如果有人认同这样的产品理念，相信通过【随波呼吸】这样一款产品能够获得健康的呼吸方法和更加美好的生活，那么他们“可以”选择我的产品。

**一些补充：**
最后，我还是想说明一下。我不认为潜在用户“一定”要选择我的产品，我也没有一个非常坚定的理由来让他们“一定”要选择我的产品。我的产品目前不是、未来也应该不太可能是市场上最棒的产品，这个世界上有太多非常优秀的团队在认真地打磨产品了。其他的产品里，也能够找到非常优质的资源来了解和练习呼吸，我的知识和经验并不是凭空而来的。

如果确实存在一个理由让用户“一定”选择某一个产品，那么这个市场应该是极其垄断的，而这个市场并不是这样的，说明在这个市场上并不存在这样的理由。另外，对于独立开发者而言，进入一个垄断市场通常也不是一个好选择。也正式因为这个市场没有“一定”的理由，像我们这样的独立开发者或小型团队，才有成长的土壤。

**Comment by ShirleneLiu** [comment]
APP 上线了的话，可以发一个体验链接过来哟
